pkg/controller/virtualmachinevolume/virtualmachinevolume_controller_test.go

```diff
@@ -54,22 +42,15 @@ var _ = Describe("Reconcile", func() {
 			for i := range volume.Status.Conditions {
 				volume.Status.Conditions[i].LastTransitionTime = v1.Time{}
 			}
-			Expect(volume.Status.Conditions).Should(Equal([]hc.VirtualMachineVolumeCondition{
-				{
-					Type:               hc.ReadyToUse,
-					Status:             corev1.ConditionFalse,
-					Reason:             "FailedCreate",
-					Message:            "VirtualMachineImage is not exists",
-					LastTransitionTime: v1.Time{},
-				},
-			}))
+			found, cond := util.GetConditionByType(volume.Status.Conditions, hc.VirtualMachineVolumeConditionReadyToUse)
+			Expect(found).Should(BeTrue())
+			Expect(cond.Status).Should(Equal(corev1.ConditionFalse))
 		})
 	})
 
 	Context("2. with false status image", func() {
 		image := newTestImage()
-		readyToUse := false
-		image.Status.ReadyToUse = &readyToUse
+		util.SetConditionByType(image.Status.Conditions, hc.ConditionReadyToUse, corev1.ConditionTrue, "VmiIsReady", "Vmi is ready to use")
```

Comment:
It is a test case where the status of the image is false, but the value put in image condition seems to be true.

Reply:
Thank you for spotting the error! It is fixed now.